code/modules/mob/living/carbon/xenomorph/castes/Sentinel.dm

```diff
@@ -19,10 +19,10 @@
 	acid_level = 1
 
 	tackle_min = 4
-	tackle_max = 4
+	tackle_max = 6
 	tackle_chance = 50
 	tacklestrength_min = 4
-	tacklestrength_max = 4
+	tacklestrength_max = 6
```

Comment:
Are you sure about the `tacklestrength_max` change? Its `rand(attacking_xeno.tacklestrength_min, attacking_xeno.tacklestrength_max)` for determining the knockdown duration for a successful tackle. Just asking if you were intentionally increasing how long they knock you over.

Reply:
If intentional, the CL needs to reflect this.